### DISCUSSION TITLE
Flexbox Media Queries

### DISCUSSION THREAD

**Discussion by leandrho** [Launch-X-Latam/MisionFrontEnd#304] [Q&A] [answered]
¿Esta disponible en alguna parte el ejemplo que se dio en clase de flexbox y media queries, es decir, los archivos html y css? Saludos

**Comment by doguedogue** [comment]
No los he visto, pero checa está página https://www.w3schools.com/css/css_rwd_mediaqueries.asp

**Comment by iam-leo** [comment, marked as answer]
Los archivos del MC no están, pero te dejo [aquí](https://css-tricks.com/snippets/css/a-guide-to-flexbox/#aa-flexbox-properties) la guía que utilizó para explicar flexbox, y para media queries dejo esta [alternativa](https://lenguajecss.com/css/responsive-web-design/media-queries/).
